modules/javafx.controls/src/test/java/test/javafx/scene/control/MenuItemTest.java

```diff
@@ -228,20 +227,14 @@
     @Test public void getUnspecifiedTextProperty1() {
         MenuItem mi2 = new MenuItem();
         assertNotNull(mi2.textProperty());
+        assertNull(mi2.getText());
     }
 
     @Test public void getUnspecifiedTextProperty2() {
         MenuItem mi2 = new MenuItem("");
         assertEquals("", mi2.getText());
     }
 
-    @Ignore // calling textProperty will no ensure text value is non null
-    @Test public void unsetTextButNotNull() {
-        MenuItem mi2 = new MenuItem();
```

Comment:
Test was incorrect. The value of the 'text' property is not set and asserting on not null is incorrect.